mods/persistence/modules/overmap/exoplanets/_exoplanet.dm

```diff
@@ -1,8 +1,10 @@
+//#TODO: Gotta fix this so we just run the exoplanet's gen code on startup instead to properly update areas and stuff for unsaved turfs.
```

Comment:
Has this been fixed?

Reply:
Nope. I just added a todo for it.

Reply:
Nope. This will require work upstream. I think loaf mentioned he'd fix planet so they init at the same time as everything else. But it probably will take a while.